### PR TITLE
Tickets/sitcom 476

### PR DESCRIPTION
**Pull request by isotuela**
Pending on filter, and conditions assessment section.

### CODE REVIEW

File: notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb [hunk 1]
```diff
@@ -0,0 +1,761 @@
+{
```
Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: <p>might be worth adding a link to the ticket.</p><p>In fact, I wonder if each notebook should have a link to the ticket in the title section just as a standard thing...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: <p>go with <code>SDSSr_65mm</code> </p><p>I commented on the jira ticket. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: <p>better to set <code>mag_limit=8</code>,then use that variable in the method call and in the log statement  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: <p>Set the <code>program</code> key as well please. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: <p>I don't think "Clear" is the right word here.  Consider:</p><p>"Move hexapod to in-focus position"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: 👍 I usually add the link to the jira ticket in the title section.

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: 👍

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: 👍

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: 👍

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb:1]: Thanks 👍

**Pull request merged**